lib/chef/knife/vsphere_vm_find.rb

```diff
@@ -0,0 +1,277 @@
+#
+# Author:: Raducu Deaconu (<rhadoo_io@yahoo.com>)
+# License:: Apache License, Version 2.0
+#
+
+require 'chef/knife'
+require 'chef/knife/base_vsphere_command'
+require 'rbvmomi'
+require 'netaddr'
+
+PsOn = 'poweredOn'
+PsOff = 'poweredOff'
+PsSuspended = 'suspended'
+
+#find vms belonging to pool that match criteria, display specified fields
+class Chef::Knife::VsphereVmFind < Chef::Knife::BaseVsphereCommand
+  banner "knife vsphere vm find"
+
+  common_options
+option :pool,
```

Comment:
The indentation seems to be out of whack here and throughout the file. Could you have a look, please, and reformat to two spaces per indentation level?